source/content/drupal-9.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Drupal 9 Early Access
+description: Test the latest Drupal on the Pantheon Website Management Platform.
+categories: [get-started]
+tags: [site]
+earlyaccess: true
+contributors: [populist, edwardangert]
+reviewed: "2020-05-21"
+---
+
+Drupal 9 is, according to [drupal.org](https://www.drupal.org/docs/understanding-drupal/drupal-9-resources), "a cleaned-up version of Drupal 8... \[with] deprecated code removed and third-party dependencies updated."
+
+D9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://symfony.com/releases/4.4) and [Twig 2](https://twig.symfony.com/doc/2.x/index.html), removes several deprecated API functions in favor of better options, and allows everyone running Drupal 8.8+ an easy upgrade path to D9 & beyond.
+
+<Alert title="Warning" type="danger">
+
+Currently, Drupal 9 is only available on Pantheon as a Early Access feature release. This means the feature is currently in active development and is intended for testing and evaluation purposes only. **Sites may be deleted without notice**.
```

Comment:
Regarding the merge conflict problem described below, that's going to happen in both directions. That is, if someone took their site from 8.8.5 or 8.8.6 (whatever we base the 9.0.0-beta3 tag on), and then later want to dashboard-update or even just git-merge-update their site from the pre-release to the stable release, they are going to get the kind of merge conflicts that -X theirs is not going to help.

We should have some sort of warning about this. We might decide to provide a conversion tool to help with this transition, but I think it would be better to impart the understanding that Drupal 9 environments are throwaway. That is, if you make a new Drupal 9 site, we may delete it. If you make a multidev environment from your production site and switch it to Drupal 9, we won't delete the whole site, but neither should we guarantee that environment is going to keep running correctly after the 9.0.0 stable release.

Reply:
I updated this copy to

```
**Do not use this, or any, Early Access release for your live site.**

Currently, Drupal 9 is only available on Pantheon as an Early Access feature release. This means the feature is currently in active development and is intended for testing and evaluation purposes only.

Pantheon engineers are rolling out changes often, and easy updates from Drupal 9 Early Access are not guaranteed.

**Sites may be deleted without notice**.
```